.github/workflows/container-tests.yml

```diff
@@ -11,6 +11,7 @@ jobs:
     runs-on: ubuntu-20.04
     strategy:
       fail-fast: false
+      container-to-test: ["postgresql-container", "s2i-nodejs-container", "s2i-python-container"]
```

Comment:
Does this work? Shouldn't it be in a `matrix`'s scope?

Reply:
Good catch separate_containers_in_ci_github_actions

Reply:
Hopefully, I will work.